action.yml

```diff
@@ -0,0 +1,101 @@
+name: "Sourcery GitHub Action"
+
+branding:
+  icon: 'box'
+  color: 'orange'
+
+description: "A GitHub Action for running Sourcery."
+
+inputs:
+  token:
+    description: "The Sourcery token for logging in."
+    required: true
+  target:
+    description: "File(s) or directory(ies) to review."
+    required: false
+    default: "."
+  version:
+    description: "Sourcery CLI version, e.g. '0.12.10'. If empty, default to the latest version."
```

Comment:
🤓 nitpick: Of course, the example should be the latest & greatest: 0.12.12 :smiley:

Reply:
I totally agree with that! I can update it now, of course - but it will surely get outdated every two weeks as we release new Sourcery versions.
I created an enhancement proposal for doing this automatically in the future: https://github.com/sourcery-ai/core/issues/2005